aio/content/marketing/resources.json

```
@@ -365,6 +365,18 @@
             "title": "Angular Material",
             "url": "https://material.angular.io/"
           },
+          "mdb5": {
+            "desc": "Free components, templates & plugins for the latest Bootstrap 5 styled with accordance to Material Design 2.0 (guidelines introduced in 2020)",
```

I am not a native speaker, but I believe this should be `in accordance with`:

```suggestion
            "desc": "Free components, templates & plugins for the latest Bootstrap 5 styled in accordance with Material Design 2.0 (guidelines introduced in 2020)",
```

(Same below.)